src/simulator.zig

```diff
@@ -26,14 +26,15 @@ var cluster: *Cluster = undefined;
 pub fn main() !void {
     // TODO Use std.testing.allocator when all deinit() leaks are fixed.
     const allocator = std.heap.page_allocator;
+    const ga = std.testing.allocator;
```

Comment:
We can use `allocator` from the previous line instead of bringing in another scope variable.